src/doctopus/files/predicates.clj

```diff
@@ -17,16 +22,18 @@
   [file-handle type-regex]
   (is-type? (.getPath file-handle) type-regex))
 
+;; ### File Predicates
+;; Used for checking that an object representing a file -- whether it's a String
+;; or a java.io.File -- has a particular extension
 
 (defn markdown?
-  "returns true only for type files."
   [file-handle]
-  (is-type? file-handle #"(?i)md|markdown|mdown$"
-            ))
-
+  (is-type? file-handle #"(?i)md|markdown|mdown$"))
```

Comment:
yeah, that's what i meant. i don't feel too strongly about it, i just feel like it models the actual problem better and is easier to provide values to and get them back. ("hey, what is a list of extensions you care about?" vs. "hey, here is a string, do you like it?"

i should stress that i really don't care too much, especially since it's just hack week, but i just felt inclined to mention it.